tests/integration/cassandra_test.go

```diff
@@ -1,408 +1,295 @@
 //go:build integration
 // +build integration
 
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package integration
 
 import (
-	"flag"
+	"context"
 	"fmt"
-	"github.com/newrelic/infra-integrations-sdk/log"
-	"github.com/newrelic/nri-cassandra/tests/integration/helpers"
 	"github.com/newrelic/nri-cassandra/tests/integration/jsonschema"
+	"github.com/newrelic/nri-cassandra/tests/integration/testutils"
 	"github.com/stretchr/testify/assert"
-	"io/ioutil"
-	"os"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/testcontainers/testcontainers-go"
 	"path/filepath"
-	"regexp"
 	"strings"
 	"testing"
 	"time"
 )
 
 const (
 	envCassandraVersion = "3.11.0"
-	timeout             = "30000"
+	timeout             = "5000"
 )
 
 var (
 	iName = "cassandra"
 
-	defaultContainer = "integration_nri-cassandra_1"
-
 	iVersion = "1.1.0"
 
-	defaultBinPath  = "/nri-cassandra"
-	defaultHostname = "cassandra"
-
-	schemaFolder = fmt.Sprintf("json-schema-files-%s", envCassandraVersion)
+	integrationBinPath       = "/nri-cassandra"
+	integrationContainerName = "integration_nri-cassandra_1"
+	cassandraContainerName   = "integration_cassandra_1"
 
-	// cli flags
-	container = flag.String("container", defaultContainer, "container where the integration is installed")
-	binPath   = flag.String("bin", defaultBinPath, "Integration binary path")
-
-	hostname = flag.String("hostname", defaultHostname, "cassandra hostname")
+	schemaDir = fmt.Sprintf("json-schema-files-%s", envCassandraVersion)
```

Comment:
why having it as a global variable?

Reply:
leftover from old code, moved as it's used in only one place